ui/plugins/failure_summary/controller.js

```diff
@@ -63,7 +63,21 @@ treeherder.controller('BugsPluginCtrl', [
                             );
                             suggestions.push(suggestion);
                         });
+                        var errors = [];
```

Comment:
In looking at this, I'm not sure we can be guaranteed that the order of the artifacts as they come back will always be the same.  It would be safer to use ``_.find`` to ensure you've got the right one.  This would be true for "Bug suggestions" as well as "text_log_summary"  Check out https://lodash.com/docs#find

We also have examples of using ``_.find`` in other places in the JS code.  Would you please alter this to use ``_.find``?